src/main.rs

```diff
@@ -219,12 +214,15 @@ struct Args {
     oci_password: Option<String>,
 }
 
+#[allow(clippy::too_many_lines)]
 #[tokio::main]
 async fn main() -> anyhow::Result<()> {
     let Args {
         log_level,
         nats_host,
         nats_port,
+        nats_jwt,
```

Comment:
This is totally a preference thing, but can we just switch this to `let args = Args::parse()` and then just use `arg.<var>` instead? This just looks kinda ridiculous at this point (and triggers a clippy lint apparent) 😆

Reply:
I don't mind either way

Reply:
Did it, can always drop the commit if we hate it.